kouncil-frontend/libs/feat-send/src/lib/send/send.component.ts

```diff
@@ -171,16 +175,25 @@ export class SendComponent {
   }
 
   onSubmit(messageData: MessageData): void {
+    this.isSendButtonDisabled = true;
     this.messageDataService.setMessageData(messageData);
     this.sendService.send$(this.servers.getSelectedServerId(), this.countControl.value, messageData)
       .pipe(first())
       .subscribe(() => {
         this.dialog.closeAll();
         this.resetForm();
+        this.isSendButtonDisabled = false;
```

Comment:
I think that it would be better to fix that case when message is sent as string rather than disabling send button, is it hard to fix that? Did you try?